pysingfel/solvent_form_factor/tests/libsaxs_compare_test.py

```diff
@@ -50,48 +42,32 @@ def setUp(self):
     def test_vacuum_form_factor_data(self):
         print "Testing vacuum form factor data against libsaxs...\n"
```

Comment:
These print statements should be changed to comments:
# Testing vacuum form factor data against libsaxs

Idea is to keep the unittest clean until a test fails

Reply:
OK, I made the statements into comments.